collect_app/src/main/res/values/strings.xml

```diff
@@ -484,8 +484,8 @@
     <string name="form_metadata_summary">These values will be added to forms that have username, email and/or telephone preloaded fields to identify the person submitting data.</string>
     <string name="phone_number">Phone number</string>
     <string name="email">Email address</string>
-    <string name="user_defined">User-defined</string>
-    <string name="device_defined">Device-defined</string>
+    <string name="user_defined">User defined</string>
```

Comment:
Hmmm yeah it feels wrong to me as a title (rather than in a sentence). Are we using that as a language guide for English? Would be good to reference that.